lib/WP_Auth0_Lock10_Options.php

```diff
@@ -11,10 +11,11 @@ class WP_Auth0_Lock10_Options {
 	/**
 	 * WP_Auth0_Lock10_Options constructor.
 	 *
-	 * @param array $extended_settings - argument in renderAuth0Form(), used by shortcode and widget
+	 * @param array                 $extended_settings Argument in renderAuth0Form(), used by shortcode and widget.
+	 * @param null|WP_Auth0_Options $opts WP_Auth0_Options instance.
 	 */
-	public function __construct( $extended_settings = array() ) {
-		$this->wp_options        = WP_Auth0_Options::Instance();
+	public function __construct( $extended_settings = array(), $opts = null ) {
```

Comment:
Dependency injection to help with testing. I would have just refactored or added a new class but not sure if Lock is sticking around in the plugin.